rpcs/fetcher_test.go

```diff
@@ -1,414 +0,0 @@
-// Copyright (C) 2019-2020 Algorand, Inc.
```

Comment:
Can this file be moved to /catchup/fetcher_test.go instead of deleting it?

Reply:
While the file has been deleted, all the test that it included were transported into the the catchup library. ( some of these have been re-written to various degree )